build/Dockerfile

```diff
@@ -1,18 +0,0 @@
-FROM golang:1.23.4-bookworm
```

Comment:
if we were not doing docker in docker how this was used?

Reply:
I don't think it was being used anywhere.